### DISCUSSION TITLE
Please tell me the purpose of rcx file

### DISCUSSION THREAD

**Discussion by chros098** [The-OpenROAD-Project/OpenROAD-flow-scripts#1464] [General]
hello
I don’t know the purpose of the rcx file and would like to ask？In which step is the rcx file used?If I ignore the rcx file variable, will it affect the wiring?

**Comment by rovinski** [comment]
RCX is RC extraction. It is used in extracting the parasitics for the routing and creating sign-off level static timing analysis. The timing before RCX is run is just an estimate.

RCX is used in the final reporting step.

It will not affect the wiring, but it will indicate whether your design meets timing and whether you need to rerun with different constraints.
